homeassistant/const.py

```diff
@@ -22,6 +22,9 @@
 # If no name is specified
 DEVICE_DEFAULT_NAME = "Unnamed Device"
 
+# Max characters for an event_type
+MAX_LENGTH_EVENT_TYPE = 64
```

Comment:
We shouldn't change the max length in this PR. That should be a separate PR. Let's just raise an exception here.

Reply:
Ah right, that's why the linked PR has to go first. Thanks for checking me on that 👍🏾

Reply:
Or are you suggesting we bump this down to 32 so we can merge and then rebase the other PR to change the limit using the constant?

Reply:
Done - Just a reminder about https://github.com/home-assistant/core/pull/48115#issuecomment-802926833 - reloading that config entry type would raise this exception if this gets released before the other PR